src/tensorflow_backend_tf.cc

```diff
@@ -209,14 +209,21 @@ IsGPUFeedAndFetchSupported(TRITONTF_DataType dtype)
 
 void
 NewSessionOptions(
-    const bool has_graph_level, const int graph_level,
-    const bool allow_gpu_memory_growth,
+    const int num_intra_threads, const int num_inter_threads,
+    const bool use_per_session_threads,
+    const std::vector<std::string>> &graph_tags,
+    const std::vector<std::string>> &signature_defs, const bool has_graph_level,
```

Comment:
when is graph_def and signature_def used here? I don't see it being passed in the calls for this function either

Reply:
@jbkyang-nvi you should remove these as well